reference/5-api-publisher/readme.md

```diff
@@ -1,9 +1,130 @@
-# API Publisher
+# Ed-Fi API Publisher
 
-:::tip
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher/badge)](https://securityscorecards.dev/viewer/?uri=github.com/Ed-Fi-Alliance-OSS/Ed-Fi-API-Publisher)
```

Comment:
The OSSF scorecard should not be part of the documentation. It only belongs in the code repository.

Reply:
removed